### PR TITLE
Ms/tc 06.003.05

### PR DESCRIPTION
**Pull request by MartaShepel**
https://trello.com/c/KKiK3X5n/596-tc0600305-organization-folder-rename-organization-folder-from-drop-down-list

### CODE REVIEW

**Review by AliaksandrMakaranka — APPROVED** [review]

**Review by Vlad1kek — APPROVED** [review]

File: src/test/java/school/redrover/OrganizationFolder9Test.java [hunk 1]
```diff
@@ -62,4 +62,35 @@ public void testCreateOrganizationFolderWithValidName () {
...
     }
+
+    final String folderName = "OrganizationFolder5";
+    final String renamedFolder = "NewOrganizationFolder555";
+    private void createFolder () {
+        getDriver().findElement(By.xpath("//a[@href='/view/all/newJob']")).click();
+        getDriver().findElement(By.xpath("//input[@id='name']")).sendKeys(folderName);
+        getDriver().findElement(By.xpath("//li[@class='jenkins_branch_OrganizationFolder']")).click();
+        getDriver().findElement(By.xpath("//button[@id='ok-button']")).click();
+        getDriver().findElement(By.xpath("//button[@name='Submit']")).click();
```
Comment (EvgenyJavaTest) [inline src/test/java/school/redrover/OrganizationFolder9Test.java:73]: 73: This step is redundant. The folder was created after the previous step

File: src/test/java/school/redrover/OrganizationFolder9Test.java [hunk 2]
```diff
@@ -62,4 +62,35 @@ public void testCreateOrganizationFolderWithValidName () {
...
+
+        getDriver().findElement(By.linkText("Dashboard")).click();
```
Comment (EvgenyJavaTest) [inline src/test/java/school/redrover/OrganizationFolder9Test.java:75]: 75: Strictly speaking, this action does not apply to creating a folder. I think that when creating other tests that contain this method, you will not need to return to the dashboard in every case.

File: src/test/java/school/redrover/OrganizationFolder9Test.java [hunk 3]
```diff
@@ -62,4 +62,35 @@ public void testCreateOrganizationFolderWithValidName () {
...
+    }
+
+    @Test
+    public void testCreateOrganizationFolderWithValidNameNew () {
+        createFolder();
```
Comment (EvgenyJavaTest) [inline src/test/java/school/redrover/OrganizationFolder9Test.java:80]: This test does not contain an Assert. He doesn't check anything. You can check that when you go to the main page, the created folder is displayed there.

**Comment by EvgenyJavaTest** [comment]
You've added two tests, but only one Trello card is attached. For each test, you need to make a separate PR with an attached Trello card.

**Pull request merged**